vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
Thanks a lot Enrico for the comment, I'll think about it.

It worries me thou that we need a special case just for Obligations (that by the way violates layering), but I don't understand things well enough yet.